__tests__/contact/contact.mutation.test.js

```diff
@@ -75,11 +77,17 @@ describe('Contacts mutations test', () => {
     ]);
     expect(res.data.addContact.address).toBeInstanceOf(Object);
     expect(res.data.addContact).toHaveProperty('email', 'test@test.com');
```

Comment:
Please, use here values from the `contact` instead of plain hardcode literals. It would be much easier and safe to changes some values in `contact` and be sure that it is changed in tests too. Use hardcode values only for properties that you cannot take from `contact`